neuralprophet/forecaster.py

```diff
@@ -3012,3 +3025,30 @@ def _reshape_raw_predictions_to_forecst_df(self, df, predicted, components):
                         yhat_df = pd.Series(yhat, name=comp).set_axis(df_forecast.index)
                         df_forecast = pd.concat([df_forecast, yhat_df], axis=1, ignore_index=False)
         return df_forecast
+
+    def conformalize(self, df_cal, alpha, method="naive", plotting_backend="default"):
```

Comment:
If a user configures QR, then uses CP, I would expect for them to receive CQR by default. 
It appears to me that the default `method="naive"` will however ignore the QR estimates. 
Maybe we could even remove the `method` arg, as QR + CP = CQR and no-QR + CP = naive CP.
Does that make sense?

Reply:
No, naive is a legitimate conformal prediction method. User should be able to use it regardless of whether there is any QR or not. So my design philosophy is that the user can use whatever CP method regardless of the trained model.

As for what the default method should be, for QR or not QR, that can be debatable. I am leaning towards creating an `"auto"`, to encompass what you said: QR + CP = CQR and no-QR + CP = naive CP.  But that can be added in a new PR.

Reply:
I would definitely love to see a more unified approach to our different uncertainty modelling approaches - it's getting rather confusing to me and also how different their APIs are or rather my uncertainty on how to use them / combine them.